commitizen-tools/commitizen
# Feature/changelog stdout

## Description

Include a new flag on `bump` to send the changelog to stdout.

This allows the incremented changelog data to be captured and sent to a file.

I decided to reuse the changelog with `--dry-run` when this flag is set, instead of adding a new flag on changelog.
I think this is not expensive because the changelog will only be generated for just a single release/tag.

## Checklist


## Expected behavior

A normal bump should happen + changelog to the stdout

## Steps to Test This Pull Request


1. Create a new commit.
2. Run `cz bump --changelog --changelog-to-stdout > body.md`
3. Observe `body.md` which should contain only the incremented changes

## Additional context


Closes  #225
